project.md

```diff
@@ -5,47 +5,168 @@
+http://taikosource.com/wp-content/uploads/2014/08/Kanji-West-Notation-090214.pdf
+^ note that this isn't entirely standardized, though. Some places say things
+differently.
+
+Kuchi-showa has its own perks (personally, it's a lot more fun to chant out the
+notes with your friends instead of silently pouring through pages of music) but
+undoubtedly, a major downside is that songs are more easily misremembered -- or
+worse, lost entirely. Thus, it would be useful to have some kind of sheet music
+notation for taiko. One might think we can directly utilize music sheet for
+regular drums, but I believe a lot of nuance would be lost. For instance, you
+would want to specify which notes are dons and which notes are kas. Furthermore,
+because taiko is a group performance, knowing what hand to strike with is also
+very important. This makes regular drum notation a bit lacking for
+translating taiko songs to sheet music.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because there is currently no specification for being able
+to create taiko sheet music. Thus, it'd be useful for some kind of taiko
+language to exist with the intent that the quirks that taiko has but other
+percussion instruments don't are communicated clearly. Additionally, by creating
+a language around taiko notes that can be written down, it may provide people
+who find learning orally difficult a different, more effective medium to learn a
+song.
+
+You could always write it out physically without the need for a program, but having
+a program that you can run has many benefits, including but not excluding easier
+method of editing old sheet music, running sheet music through some interpreter
+to play out the piece, and more.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I've been in the taiko club at the Claremont Colleges since sophomore year and
+it's honestly one of the highlights of my week. Hitting drums relieves your
+stress like nothing else. 
+
+Our club is pretty small; there's only a handful of people from each grade and
+when the seniors graduate, it's a major loss. One of these losses involve not
+knowing how exactly a song goes because we were taught orally by the
+seniors. There are videos of past performances we can look at, but the sound
+quality isn't the greatest nor can we trust that the performers themselves did
+it right. Additionally, we can't really google any of these because again, these
+songs are passed down from instructor to student orally.
+
+Just the other week, we had a minor incident where no one could agree how
+exactly a song went. We ended up deciding on something, but it's very possible
+that we were not correct and therefore altered how the song originally
+went. Thus, I think it would be very helpful to have some kind of physical aid
+to remember how a song goes instead of relying on faulty memory.
+
```

Comment:
This is really great motivation for your project! Definitely seems like it would serve a vital need for the club, as well as giving new learners additional tools to pick up taiko